src/_pytest/assertion/rewrite.py

```diff
@@ -51,19 +51,6 @@ def ast_Call(a, b, c):
         return ast.Call(a, b, c, None, None)
 
 
-def ast_Call_helper(func_name, *args, **kwargs):
```

Comment:
for that one i'd like to know why it was and why it isnt

Reply:
last usage removed in 3e6f1fa2db20eb2fd751a778dc42dd126a9e778d